support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -33,6 +33,7 @@ object CheckoutValidationRules {
 
   def validate(createSupportWorkersRequest: CreateSupportWorkersRequest): Result =
     (createSupportWorkersRequest.product match {
+      case _: SupporterPlus => PaidProductValidation.passes(createSupportWorkersRequest)
```

Comment:
this should be the same as contributions but I notice contributions validation was never switched on!  So this is an improvement.